.github/workflows/build-reuse-win.yml

```diff
@@ -56,6 +56,10 @@ on:
         required: false
         default: ''
         type: string
+      official:
```

Comment:
I still find "official" to be a confusing term, especially in GitHub, which should never build production/shipping/signed bits.